### PR TITLE
doc: update docs on groups and roles

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] Migration step added in case of breaking change
- [ ] User documentation updated
- [ ] (If you have changed REST API interface) view-swagger.ftl updated
- [ ] Test plan template updated
- [ ] Clean commits
- [ ] Added Feature/Fix to release notes

### CODE REVIEW

**Comment by fdlk** [comment]
[Update MOLGENIS docs on OIDC](https://trello.com/c/0oV05Iol/802-update-molgenis-docs-on-oidc)

**Review by sidohaakma — COMMENTED** [review]
Just to check. Your argument not to add images is that we would need to bootstrap all images again on the docs?
Maybe I can do this? To align with the current styling?

**Comment by fdlk** [comment]
A couple of issues with the screenshots:
* They need replacing whenever the screen or style changes
* The UI has a lot of simple screens so you need many screenshots of all the steps that are by themselves very simple
* It's currently hard to see what's documentation and what's screenshots:
Schermafbeelding 2021-03-19 om 22 04 43

File: docs/guide-authentication.md [hunk 1]
```diff
@@ -100,6 +100,22 @@ MOLGENIS requires at least the `sub` and `email` claims and ideally the `given_n
...
+
+You can provide a json path (typically `eduperson_entitlement`), that will be evaluated on the combined claims
+of the id token and the response of the user info endpoint.
+The values in this claim will be added to the VO group table.
+In the Security Manager, group managers can grant roles on their group to
+VO group members. VO Group members will be granted these group roles when
+they log in.
+
 #### Endpoints
 The identity provider specifies a set of endpoints where the user and MOLGENIS
```
Comment (sidohaakma) [inline docs/guide-authentication.md:120]: Do you want to update this as well?

**Review by sidohaakma — APPROVED** [review]

**Review by tommydeboer — APPROVED** [review]

File: docs/guide-authentication.md (hunk 1 above)
Comment (fdlk) [reply docs/guide-authentication.md:120]: I'll make new PR, let's merge this

**Pull request merged**